experiments/README.md

```diff
@@ -14,6 +14,8 @@ This directory contains replication code for
 
 * Wager and Athey (2018): This paper is not based on GRF, but on the deprecated `causalForest`. For replication code see https://github.com/swager/causalForest
 
+* Yadlowsky, Fleming, Shah, Brunskill, and Wager (2021): The method is available in the GRF function `rank_average_treatment_effect`. For replication code see https://github.com/som-shahlab/risk-vs-hte
```

Comment:
I should create a new, clean, and targeted repo with just the code that went into the final experiments that are used in the paper. I'll take that as a to-do on my end. Maybe https://github.com/som-shahlab/rank-weighted-average-treatment-effects-paper?

Reply:
Sounds good, let me know what repo name you want, https://github.com/som-shahlab/rate-paper sounds like simpler url